Failure on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34566&attempt_id=0a169ec5-960c-47b6-99cb-a0ece1006e7e)

<details><summary>Partial log (click to expand)</summary><p>

```
make: *** [Makefile:522: moc_settings.o] Error 1
g++: fatal error: no input files
compilation terminated.
make: *** [Makefile:525: moc_database.o] Error 1
g++: error: moc_application.cpp: No such file or directory
g++: fatal error: no input files
compilation terminated.
make: *** [Makefile:543: moc_application.o] Error 1
builder for '/nix/store/z064bqkj54dysi6r5cf9n9r78k5dm1cj-QMediathekView-2017-04-16.drv' failed with exit code 2
error: build of '/nix/store/z064bqkj54dysi6r5cf9n9r78k5dm1cj-QMediathekView-2017-04-16.drv' failed
```
</p></details>

--- a/pkgs/applications/video/qmediathekview/default.nix
+++ b/pkgs/applications/video/qmediathekview/default.nix
@@ -1,0 +1,33 @@
+{ stdenv, fetchFromGitHub, qtbase, qttools, xz, boost, qmake, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "QMediathekView";
+  version = "2017-04-16";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "adamreichold";
+    repo = pname;
+    rev = "8c69892b95bf6825bd06a8c594168a98fe7cb2d1";
+    sha256 = "1wca1w4iywd3hmiwcqx6fv79p3x5n1cgbw2liw3hs24ch3z54ckm";
+  };
+
+  postPatch = ''
+    substituteInPlace ${pname}.pro \
+      --replace /usr ""
+  '';
+
+  buildInputs = [ qtbase qttools xz boost ];
+
+  nativeBuildInputs = [ qmake pkgconfig ];
+
+  installFlags = [ "INSTALL_ROOT=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "An alternative Qt-based front-end for the database maintained by the MediathekView project";
+    inherit (src.meta) homepage;
+    license = licenses.gpl3Plus;
+    platforms = [ "i686-linux" "x86_64-linux" ];
+    maintainers = with maintainers; [ dotlambda ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -16781,6 +16781,8 @@ with pkgs;
 
   qmapshack = libsForQt5.callPackage ../applications/misc/qmapshack { };
 
+  qmediathekview = libsForQt5.callPackage ../applications/video/qmediathekview { };
+
   qmetro = callPackage ../applications/misc/qmetro { };
 
   qmidinet = callPackage ../applications/audio/qmidinet { };

